cos_registration_agent/cos_registration_agent.py

```diff
@@ -42,16 +42,17 @@ def __init__(self, cos_server_url: str, device_id: str):
             logger.error(error_message)
             raise RuntimeError(error_message)
 
-    def register_device(self, **fields: Union[str, Set[str]]) -> None:
```

Comment:
Why remove the return type ?

Reply:
ah in the previous implelementation i was returning the response and when I changed it i forgot to readd the None. Fixed in [ab714d1](https://github.com/canonical/cos-registration-agent/pull/53/commits/ab714d14c07e5f357697e4f86d647f675d2fa9b2)